api/app/settings/common.py

```diff
@@ -472,6 +475,11 @@
         "LOCATION": CHARGEBEE_CACHE_LOCATION,
         "TIMEOUT": 12 * 60 * 60,  # 12 hours
     },
+    ENVIRONMENT_DOCUMENT_CACHE_LOCATION: {
+        "BACKEND": "django.core.cache.backends.db.DatabaseCache",
```

Comment:
I wasn't entirely sure whether it's better to use the database cache backend here or just the local memory. I went with database as I figured it would be better to share it between the instances and we're still reducing multiple intensive queries down to a single query. ... but I'm interested to know your thoughts @gagantrivedi @dabeeeenster

Reply:
Yep 100% db backed

Reply:
Yep, It should be db backed